### DISCUSSION TITLE
Count

### DISCUSSION THREAD

**Discussion by kazuser** [SchneiderInfosystems/FB4D#76] [Q&A] [answered]
Hi!

Is there any way to count all the childs?

![xs2ir](https://user-images.githubusercontent.com/3335273/113469253-04a27080-946e-11eb-8fa3-44ff0a4a77de.png)

**Comment by SchneiderInfosystems** [comment, marked as answer]
There is unfortunately no RecordCount because Firebase is a distributed database.

For more information see also:
https://stackoverflow.com/questions/15148803/in-firebase-is-there-a-way-to-get-the-number-of-children-of-a-node-without-load
